src/xdocs/checks/misc/avoidescapedunicodecharacters.xml

```diff
@@ -79,10 +79,14 @@
           Examples of using Unicode:
         </p>
         <source>
-String unitAbbrev = &quot;&#x3bc;s&quot;;     // OK, perfectly clear even without a comment.
-String unitAbbrev = &quot;\u03bcs&quot;;// violation, the reader has no idea what this is.
-return '\ufeff' + content;    // OK, an example of non-printable,
-                              // control characters (byte order mark).
+public class Example1 {
+  String unitAbbrev = &quot;&#x3bc;s&quot;;
+  String unitAbbrev2 = &quot;\u03bcs&quot;;// violation
```

Comment:
Please restore all ok comments and explanation wording for violation comments.

Reply:
sure @romani sir, I will take care of comments as well from the next time. And I do these changes now.